package.json

```diff
@@ -42,7 +42,8 @@
     "@remix-run/node": "2.15.0",
     "@remix-run/react": "2.15.0",
     "@supabase/supabase-js": "2.48.1",
-    "@tanstack/react-query": "5.62.8",
+    "@tanstack/react-query": "5.65.1",
+    "@tanstack/react-query-devtools": "5.65.1",
```

Comment:
I added dev tools because it can be very nice for debugging react query stuff. It doesn't affect the production bundle though because by default it is included only when app is running in the development mode.

I also bumped the query version along the way